astropy/units/format/base.py

```diff
@@ -73,15 +73,34 @@ def _format_unit_list(cls, units):
         )
 
     @classmethod
-    def _format_fraction(cls, scale, numerator, denominator):
+    def _format_fraction(cls, scale, numerator, denominator, *, fraction="inline"):
+        if not (fraction is True or fraction == "inline"):
+            raise ValueError(
+                "format {cls.name!r} only supports inline fractions,"
+                f"not fraction={fraction!r}."
+            )
+
         if cls._space in denominator:
             denominator = f"({denominator})"
         if scale and numerator == "1":
             return f"{scale}/ {denominator}"
         return f"{scale}{numerator} / {denominator}"
 
     @classmethod
-    def to_string(cls, unit, inline=False):
+    def to_string(cls, unit, *, fraction=True):
+        """Convert a unit to its string representation.
+
+        Parameters
+        ----------
+        unit : |Unit|
+            The unit to convert.
+        fraction : {False|True|'inline'|'display'}, optional
+            If `False`, display any unit bases raised to negative powers as
+            they are (e.g., ``km s^-1``).  If not `False`, use a fraction
+            instead (e.g., ``km / s``), either inline or with a multiline
+            display (if the format class supports it). If `True`, use the
+            default fraction style.
+        """
```

Comment:
'display' comes from `latex`. While for `latex` it is not necessarily multiline, I agree that is clearer here, so let's go with that.

Also, on second thought I had decided to make `True` and 'inline' the same rather than go for a class default. But it seems I forgot to update the docstring here, sorry! In any case, would that remove the confusion?

Reply:
I'd say yes. This should also changed in `core.py`//**tostring(…kwargs)**.

Reply:
Adapted (and checked that the formatting looks good in sphinx)